distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -60,8 +60,8 @@ grant codeBase "file:/org.apache.tika.core/catalog-core-urlresourcereader" {
     // Add required URL Resource Reader permissions here.
 
     // Example:
-    // permission java.io.FilePermission "/test/some/directory", "read";
-    // permission java.io.FilePermission "/test/some/directory${/}-", "read";
+    // permission java.io.FilePermission "${/}test${/}some${/}directory", "read";
```

Comment:
You should still be able to configure permissions with `/`, but using `${/}` makes it consistent with how we add permissions elsewhere.